<launch>

<node
    pkg="action_sample"
    type="DoDishes_client"
    name="do_dishes_client"
    required="true"
    output="screen"
/>

<node
    pkg="action_sample"
    type="DoDishes_server"
    name="do_dishes_server"
    respawn="true"
    output="screen"
/>

</launch>